autoptsclient-zephyr.py

```diff
@@ -78,7 +69,7 @@ def cleanup(self):
 def main():
     """Main."""
 
-    client = ZephyrClient(get_iut, "zephyr_")
+    client = ZephyrClient(get_iut, "zephyr")
```

Comment:
ZephyrClient constructor should pass "zephyr" parameter. Same with Mynewt and Bluez.